### DISCUSSION TITLE
Support AVR DA, DB, DD devices

### DISCUSSION THREAD

**Discussion by wimalopaan** [modm-io/modm#1161] [Ideas]
Is / will be support for these devices included?

**Comment by salkinium** [comment]
Oh, I didn't know there are new AVR families! 
- [AVR DA family](https://www.microchip.com/en-us/products/microcontrollers-and-microprocessors/8-bit-mcus/avr-mcus/avr-da)
- [AVD DB family](https://www.microchip.com/en-us/products/microcontrollers-and-microprocessors/8-bit-mcus/avr-mcus/avr-db)

We don't have plans to add new AVRs ourselves, but we can guide you through the process if you want to do it.
